main.js

```diff
@@ -114,15 +117,21 @@ function firstInstallTour(win) {
     };
 
     progressButton.onclick = (e) => {
+      if (e.which !== 1) { //will only accept left click...
+        return;
+      }
       panel.hidePopup();
       outerbox.removeChild(dismissLabel);
-      document.getElementById('side-tabs-button').onclick(e); //will only accept left click...
+      sidetabsbuttonClick(e);
```

Comment:
I dunno, I guess if the `sidetabsbutton`'s `onclick` doesnʼt use `this`, then we donʼt need to…